eval/eval.py

```diff
@@ -152,6 +157,10 @@
 
 CatResultDict = Dict[str, Union[int, float, str]]
 
+settings = Settings()
+settings.read("../src/reynir_correct/config/GreynirCorrect.conf")
```

Comment:
A bit suspicious, seems to depend on the cwd. But if settings.read automatically applies the __FILE__ variable, to make the config file location relative to the source file itself, that would be OK.

Reply:
Fixed